api/app/src/routes/user.ts

```diff
@@ -154,14 +153,15 @@ async function revokeToken(req: Request, res: Response) {
 * Revoke access to a provider
 *
 * @param   {string}  req.query.provider    The provider to revoke access.
-* @param   {string}  req.query.userId      The internal user ID.
+* @param   {string}  req.params.userId      The internal user ID.
 
 * @return  {{success: boolean}}      If successfully removed.
 */
 router.delete(
   "/:id/revoke",
   asyncHandler(async (req: Request, res: Response) => {
-    return revokeToken(req, res);
+    const userId = getFromParamsOrFail("id", req);
```

Comment:
Ideally we would keep using `getUserIdFrom()` because it also checks its a valid UUID.

Reply:
had to change :id to :userId to be consistent with other endpoints in the file